src/panel/category/CategoryPanel.jsx

```diff
@@ -73,7 +73,7 @@ export default class CategoryPanel extends React.Component {
       return;
     }
 
-    if (window.map.mb.isMoving()) {
+    if (window.map.mb.isMoving && window.map.mb.isMoving()) {
```

Comment:
This test is required for now because `isMoving` isn't included in the Mapbox-GL-JS mock.

Reply:
Side note: this syntax could also be used, but I am not sure I like it:
```suggestion
    if (window.map.mb.isMoving?.()) {
```

Reply:
Yeah, I hesitated, not sure I'm a fan either, but it's probably a question of getting used to it.
I think the best way is still to avoid cases were we need that :)